lib/builders/testArgBuilder.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (value) => {
+    let args = [];
+    if(!value){
```

Comment:
good catch! I have jshint, but I don't believe its setup to flag these kinds of issues.